runtime/server/mcp_server.go

```diff
@@ -36,7 +36,7 @@ In the workflow, do not proceed with the next step until the previous step has b
 If a response contains an "ai_instructions" field, you should interpret it as additional instructions for how to behave in subsequent responses that relate to that tool call.
 `
 
-func (s *Server) newMCPHandler() http.Handler {
+func (s *Server) createMCPServer() *server.MCPServer {
```

Comment:
nit: `newMCPServer` – it's just convention, but "new" implies creating an in-memory object, "create" implies creating a persistent (DB-backed) object